tnefparse/tnef.py

```diff
@@ -43,10 +43,10 @@ def name_str(self):
         return TNEF.codes.get(self.name)
 
     def __str__(self):
-        return "<%s '%s'>" % (self.__class__.__name__, self.name_str)
+        return f"<{self.__class__.__name__} '{self.name_str}'>"
 
```

Comment:
I like f-strings in general, but I am not really sure f-strings are an improvement in cases such as this? It seems to me  it can be harder to get an idea what the string is about since the "structure" and the variables that go there are both 'mixed' inside f-strings. Especially if the variable names are long or otherwise 'magical'. I am fine merging as-is but just thought I'd raise the question.

Reply:
I have to admit, I still like %-formatting a lot, but I am also a big fan of consistency, and I think in the long term it is easier to go with f-strings, especially as there are tools out there helping to reach that consistency, like `pyupgrade`.

I put the f-string formatting commit at the end, so it can be easily reverted.

It is your decision, and I am fine with either way.